### DISCUSSION TITLE
Could not find haxelib "hxCodec"

### DISCUSSION THREAD

**Discussion by fianalins** [FunkinCrew/Funkin#2819] [Compiling help] [answered]
When I try to compile the game - 0.4.1 for Windows - using FunkinCrew's hxCodec (the one that hmm installs) it tells me that it is missing

![image](https://github.com/FunkinCrew/Funkin/assets/87100196/defce795-eeca-4a52-b72c-fd72cbc3e8f3)

even though I have it installed,

![image](https://github.com/FunkinCrew/Funkin/assets/87100196/a78087f2-b6c6-4f48-8ef8-f52f7346d608)

and if I use the default hxCodec, it tells me `hxcodec.flixel.FlxVideoSprite has no field autoPause`.

![image](https://github.com/FunkinCrew/Funkin/assets/87100196/4c23fa3f-78b1-40a2-9498-e82cc0499196)

I tried installing it without the hmm command and it still tells me its missing, and there doesn't seem to be any other posts about this, so what do I do from here?

**Comment by NotHyper-474** [comment]
Try this:

1. Open the Terminal in Funkin folder and run `haxelib libpath hxCodec` and copy the path that gets printed.
2. Run `haxelib --global dev hxCodec "the/path/you/copied"`

**Reply by fianalins** [reply]
I did that and it is now telling me `Type not found : T`

![image](https://github.com/FunkinCrew/Funkin/assets/87100196/e4bb03db-795b-4c3d-8892-8e421655e40c)

There is another discussion about it but it hasn't been answered. I tried uninstalling hxCodec, then doing the same thing and I get the same error.

**Reply by NotHyper-474** [reply, marked as answer]
**(If you came here looking for a fix for the hxCodec error then refer to https://github.com/FunkinCrew/Funkin/discussions/2819#discussioncomment-9782786)**

According to #2333 it seems to be an issue with Flixel, try reinstalling it via hmm reinstall -f flixel

Not sure if this will work since other guy reported it didn't change anything

If this doesn't work maybe a clean build could do the job

**Reply by fianalins** [reply]
It is giving me the same error the author got in the post.

![image](https://github.com/FunkinCrew/Funkin/assets/87100196/b77a4967-9d3f-40e9-9639-edff0b29494f)

Reinstalling everything again somehow worked though? A bunch of errors have been making it seem like the base game was going to be impossible to actually compile, especially since I have tried both builds on two seperate computers, but now it finally works, and I don't even really know what I did other than reinstall everything.

**Reply by NotHyper-474** [reply]
Maybe it was just a bad hmm install, no idea why it happens but it happens
